src/bitmessagemain.py

```diff
@@ -334,40 +368,39 @@ def start(self):
 
             connectToStream(1)
 
-            if BMConfigParser().safeGetBoolean('bitmessagesettings','upnp'):
+            if BMConfigParser().safeGetBoolean(
+                    'bitmessagesettings', 'upnp'):
                 import upnp
                 upnpThread = upnp.uPnPThread()
                 upnpThread.start()
         else:
             # Populate with hardcoded value (same as connectToStream above)
             state.streamsInWhichIAmParticipating.append(1)
 
-        if daemon == False and state.enableGUI: # FIXME redundant?
-            if state.curses == False:
-                if not depends.check_pyqt():
-                    sys.exit(
-                        'PyBitmessage requires PyQt unless you want'
-                        ' to run it as a daemon and interact with it'
-                        ' using the API. You can download PyQt from '
-                        'http://www.riverbankcomputing.com/software/pyqt/download'
-                        ' or by searching Google for \'PyQt Download\'.'
-                        ' If you want to run in daemon mode, see '
-                        'https://bitmessage.org/wiki/Daemon'
-                        'You can also run PyBitmessage with'
-                        ' the new curses interface by providing'
-                        ' \'-c\' as a commandline argument.'
-                    )
-
+        if not state.enableGUI:
+            BMConfigParser().remove_option('bitmessagesettings', 'dontconnect')
+        elif daemon is False:
```

Comment:
elif not daemon:

Reply:
This place is controversial and changes is not final. It depends on how these changes will be merged with #1221. Maybe this statement will be more like
```python
if not state.enableGUI:
    ...
elif daemon:
    ...
else:
    if state.curses:
        ...
```